tests/benchmarks/test_work_stealing.py

```diff
@@ -10,14 +10,18 @@
 from packaging.version import Version
 from tornado.ioloop import PeriodicCallback
 
+from ..utils_test import run_up_to_nthreads
 
+
+@run_up_to_nthreads("small_cluster", 50, reason="fixed dataset")
 def test_trivial_workload_should_not_cause_work_stealing(small_client):
     root = delayed(lambda n: "x" * n)(utils.parse_bytes("1MiB"), dask_key_name="root")
     results = [delayed(lambda *args: None)(root, i) for i in range(10000)]
     futs = small_client.compute(results)
     small_client.gather(futs)
 
 
+@run_up_to_nthreads("small_cluster", 50, reason="fixed dataset")
 @pytest.mark.xfail(
```

Comment:
Since we are working with versions  `> 2022.11.1`  should we remove this. We can do it in a separate PR if it's out of scope.

Reply:
Yes definitely out of scope